File: Python/Tutorial/code with harry/practice/chapter7/problem4.py
num = int(input("enter a number:  "))
prime = True
for i in range(2, num):
    if (num % i == 0):
        prime = False
        break
if prime:
    print(str(num) + " is a prime number ")
else:
    print(str(num) + " is not a prime number ")

File: C Programming/code with harry/practice/chapter6/problem4.c
#include <stdio.h>
int ten_time(int *a)
{
    return 10 * (*a);
}
int main()
{
    int num, b;
    printf("enter a number: \n");
    scanf("%d", &num);
    b = ten_time(&num);
    printf("the ten time value of number is: %d", b);

    return 0;
}